app/models/lookup_key.rb

```diff
@@ -86,16 +57,12 @@ def self.find_by_name(str)
     nil
   end
 
-  def to_label
-    "#{audit_class}::#{key}"
-  end
-
-  def is_smart_variable?
-    puppetclass_id.to_i > 0
+  def audit_class
```

Comment:
I guess this one is not necessary? `self` for a non static method is going to be an instance of the object, not `LookupKey`